wds-blocks.php

```diff
@@ -90,3 +90,26 @@ function register_block() {
 	}
 }
 add_action( 'init', __NAMESPACE__ . '\register_block' );
+
+/**
+ * Adds a WDS Block category to the Gutenberg category list.
+ *
+ * @param array  $categories The existing categories.
+ * @param object $post The current post.
+ * @return array The updated array of categories.
+ * @author WebDevStudios
+ * @since 2.0.0
```

Comment:
This may just be personal preference, but the format for most projects I've worked on has been:
```
@author
@since
[ blank line ]
@param
@return
```

Reply:
I'll fix this up. This was a direct copy and paste from the `wd_s` theme.

Reply:
🤷 well I could be completely wrong on both these then! Let's see if @gregrickaby has any thoughts?

Reply:
🤔 We do not have a standard for how these should be formatted. 

I went down the rabbit hole on this, and cannot find on the Internet a "standard" way of formatting the gaps/order either. The closest thing I've found, was the official spec which contains [an example](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#54-examples):

```php
/**
 * This is a Summary.
 *
 * This is a Description. It may span multiple lines
 * or contain 'code' examples using the _Markdown_ markup
 * language.
 *
 * @see Markdown
 *
 * @param int        $parameter1 A parameter description.
 * @param \Exception $e          Another parameter description.
 *
 * @\Doctrine\Orm\Mapper\Entity()
 *
 * @return string
 */
function test($parameter1, $e)
{
    ...
}
```

With all this said, I'm with @ravewebdev (and PSR/FIG) and prefer the gaps.